scormxblock/static/js/src/scormxblock.js

```diff
@@ -1,394 +1,42 @@
-        //           'X-CSRFToken': csrftoken
-        //         },
-        //         body: JSON.stringify(pendingValues),
-        //         credentials: 'same-origin',
-        //         keepalive: true
-        //     })
-        //       .then(response => {
-        //         if (response.ok) {
-        //           return response.json();
-        //         }
-        //       })
-        //       .then(data => {
-        //         if (typeof data['scorm_score_value'] !== "undefined") {
-        //           $(".lesson_score", element).html(data['scorm_score_value']);
-        //         }
-        //         $(".success_status", element).html(data['scorm_status_value']);
-        //       });
-        //     initPendingValues();
-        //     return 'true';
-        // } else if (CheckSafariMobile()) {
-        //     const csrftoken = GetCookie('csrftoken');
-        //     pendingValues['csrfmiddlewaretoken'] = csrftoken;
-        //     var params = new URLSearchParams(pendingValues);
-        //     navigator.sendBeacon(ios_commitUrl, params);
-        //     setTimeout(function(){ syncScoreValue()},2000);
-        //     setTimeout(function(){ syncScoreValue()},5000);
-        //     setTimeout(function(){ syncScoreValue()},10000);
-        //     initPendingValues();
-        //     return 'true';
-        // } else {
-        //     $.ajax({
-        //         type: "POST",
-        //         url: commitUrl,
-        //         data: JSON.stringify(pendingValues),
-        //         async: false,
-        //         success: function (response) {
-        //             if (typeof response['scorm_score_value'] !== "undefined") {
-        //                 $(".lesson_score", element).html(response['scorm_score_value']);
-        //             }
-        //             $(".success_status", element).html(response['scorm_status_value']);
-        //         }
-        //     });
-        //     initPendingValues();
-        //     return 'true';
-        // }
-    }    
-
-    function initPendingValues(){
-        pendingValues = getPackageData();
-    }
-
-
-    function GetLastError() {
-        // console.log(version + ' GetLastError');
-        return 0;
-    }
-
-    function GetErrorString(errCode) {
-        // console.log(version + ' GetErrorString: ' + errCode);
-        return '';
-    }
-
-    function GetDiagnostic(errCode) {
-        // console.log(version + ' GetDiagnostic: ' + errCode);
-        return 'true';
-    }
-
-
-    function SCORM_12_API() {
-        this.LMSInitialize = Initialize;
-        this.LMSFinish = Terminate;
-        this.LMSGetValue = GetValue;
-        this.LMSSetValue = SetValue;
-        this.LMSCommit = Commit;
-        this.LMSGetLastError = GetLastError;
-        this.LMSGetErrorString = GetErrorString;
-        this.LMSGetDiagnostic = GetDiagnostic;
-    }
-
-    function SCORM_2004_API() {
-        this.Initialize = Initialize;
```

Comment:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/26813045/207574954-d9a8f401-04f9-4077-94d7-bd8fb96cc514.png">

 - Got this warning. But I'm still looking to it.  
 - [Doc] https://rusticisoftware.com/resources/how-to-fix-a-missing-api-adapter-scorm-error/?utm_source=google&utm_medium=natural_search

Reply:
https://github.com/Learningtribes/edx_xblock_scorm/pull/29/commits/5dd0d77b761aa0fbb5c7bf591c299f4a37dca7af